lib/streamlit/web/server/server.py

```diff
@@ -101,9 +102,24 @@ def start_listening(app: tornado.web.Application) -> None:
     port.  It will error after MAX_PORT_SEARCH_RETRIES attempts.
 
     """
+    ssl_options = None
+    cert_file = config.get_option("server.sslCertFile")
+    key_file = config.get_option("server.sslKeyFile")
+    if cert_file and key_file:
+        ssl_ctx = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+        ssl_ctx.load_cert_chain(cert_file, key_file)
+        ssl_options = ssl_ctx
+    if sum(map(bool, [cert_file, key_file])) == 1:
```

Comment:
Nit: wow about:

```
if bool(cert_file) != bool(key_file):
```
this will spare us using `sum`, `map` and a "magic" constant `1`.

Reply:
Good idea. Thanks!